ui/views_test.py

```diff
@@ -208,7 +208,7 @@ def test_roles_setting(self):
             {
                 'program': role.program.id,
                 'role': role.role,
-                'permissions': [key for key, value in available_perm_status(profile.user).items()],
+                'permissions': [key for key, value in available_perm_status(profile.user).items() if value is True],
```

Comment:
This test failed after adding the extra permissions - it was trying to check to make sure the response was the same as expected, and the return response has `if value is True` as part of it.  The reason it wasn't failing earlier is because for both roles right now, all of the permissions are True.